src/Icon/README.md

```diff
@@ -0,0 +1,36 @@
+# Icon
+To implement Icon component into your project you'll need to check your icon in the [list of all icons](../icons/icons.md) (*if you don't know the name*) and add an import of the icon name you want to use e.g.:
+```jsx
+import Airplane from "@kiwicom/orbit-components/lib/icons/Airplane";
+```
+or if you want to import more icons at once:
+```jsx
+import { Airplane, ArrowUp, ChevronDown } from "@kiwicom/orbit-components/lib/icons";
+```
+After adding import into your project you can use it simply like:
+```jsx
+<Airplane />
+<ArrowUp />
+<ChevronDown />
+```
+## :rocket: Props
+Table below contains all type of the props available in Icon component.
```

Comment:
These props are for all generated icons, not for `<OrbitIcon />` because you are not importing it.